Readme: add cernvm forum and github issue (#3298)
cvmfs/cvmfs: README.md

    and how to test and debug.
- [cvmfs-contrib](https://cvmfs-contrib.github.io/)
  - Community-contributed packages related to CernVM-FS but not maintained by
    the CernVM-FS developer team.

> ## How to Interact With Us

Reviewer: ```suggestion

## How to get in touch with us
```

> - [CernVM Forum](https://cernvm-forum.cern.ch/) For any kind of support 

Reviewer: ```suggestion
- [CernVM Forum](https://cernvm-forum.cern.ch/) For support 
```

  questions, or problems you encounter.

> - [GitHub Issues](https://github.com/cvmfs/cvmfs/issues) For any kind of

Reviewer: ```suggestion
- [GitHub Issues](https://github.com/cvmfs/cvmfs/issues) For
```

>   bug reporting or improvement requests. This issue tracker is used for all

Reviewer: ```suggestion
  bug reports or feature requests. This issue tracker is used for all
```
